Bottom: 7e13df89f7d32e32714432faab58491d69d1a1ff
Top:    21a9c44f62e4279bdfae35f30b25fba706bb054f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-07 02:38:41 +0000

Refresh of use-tags-on-circular-contours-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 9af1946..26fff1b 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -573,7 +573,6 @@ hidgl_fill_polygon (int n_coords, int *x, int *y)
 }
 
 void
-<<<<<<< current
 tesselate_contour (GLUtesselator *tobj, PLINE *contour, GLdouble *vertices,
                    double scale)
 {
